scripts/rebuild-deb.sh

```diff
@@ -116,7 +116,9 @@ cp ./default/src/app/logproc/logproc.exe "${BUILDDIR}/usr/local/bin/coda-logproc
 cp ./default/src/app/runtime_genesis_ledger/runtime_genesis_ledger.exe "${BUILDDIR}/usr/local/bin/coda-create-genesis"
 
 mkdir -p "${BUILDDIR}/usr/lib/systemd/user"
+mkdir -p "${BUILDDIR}/mkdir -p "${BUILDDIR}/usr/lib/systemd/journald.conf.d"
```

Comment:
what is /mkdir for? Seems like this line should just be `mkdir -p "${BUILDDIR}/usr/lib/systemd/journald.conf.d"`

Reply:
Yes thank you thats an error. Should I just remove this and resubmit to the branch you suggest?